apps/docs/content/components/select/colors.ts

```diff
@@ -51,7 +39,7 @@ export default function App() {
           className="max-w-xs"
         >
           {animals.map((animal) => (
-            <SelectItem key={animal.value} value={animal.value}>
+            <SelectItem key={animal.key}>
```

Comment:
There is no need to give `value` to `SelectItem`, so it is removed.
See: https://nextui.org/docs/components/select#selectitem-props